Add information on extension footprint analyzer (#8962)
magento/devdocs: src/marketplace/sellers/code-sniffer.md

Code Sniffer is a static test that uses static code analysis to detect violations of the [Magento Coding Standard](https://github.com/magento/magento-coding-standard/) to prevent common coding errors.

> Limited information, like number of errors, warnings and coding standards version used are displayed in the Quality Report section of product lising page on the Magento Marketplace.

Reviewer: ```suggestion
Limited information, like the number of errors, warnings and the coding standards version used are displayed in the Quality Report section of product listing page on the Magento Marketplace.
```

> Limited information, like the number of errors, warnings and the coding standards version used are displayed in the Quality Report section of product listing page on the Magento Marketplace.

Reviewer 2: ```suggestion
Limited information is displayed in the Quality Report section of the product listing page on the Magento Marketplace, including the number of errors, warnings, and the coding standards version that was used.
```
